### PR TITLE
Tooltip was being included on all pages without any reason.

### PR DESCRIPTION
**Pull request by bordoni**
A customer reported on the WordPress.org forums that a couple of our scripts were being included on all pages without reason.

https://wordpress.org/support/topic/loads-on-every-page-slows-website/
https://wordpress.org/support/topic/scripts-loading-on-every-page-bloatware/

Hopefully this addresses the problem for two of the files mentioned.

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Based on usage of this asset, can we get verification that this asset still gets loaded where it's intended to and not everywhere else?

**Comment by bordoni** [comment]
@sc0ttkclark yes for sure, didn't have time to get that in place yet. Was trying to get the PR up as early as possible. We might need to run an in Branch QA but I am not sure.

File: readme.txt [hunk 1]
```diff
@@ -25,6 +26,7 @@
 * Tweak - Improve regular expressions used to parse UTC timezones by removing non-required grouping and characters. [TCMN-68]
 * Language - 0 new strings added, 0 updated, 1 fuzzied, and 0 obsoleted
 
+
```
Comment (sc0ttkclark) [inline readme.txt:29]:
```suggestion
```

File: src/Tribe/Service_Providers/Tooltip.php [hunk 2]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.8
 	 */
 	public function register() {
-		tribe_singleton( 'tooltip.view', '\Tribe\Tooltip\View' );
+		tribe_singleton( 'tooltip.view', View::class );
 
-		$this->hook();
+		$this->register_assets();
```
Comment (sc0ttkclark) [inline src/Tribe/Service_Providers/Tooltip.php:24]: Just to confirm, it's intentional to also not run this on the tribe_common_loaded action, correct?

**Review by sc0ttkclark — COMMENTED** [review]
Qs

File: src/Tribe/Service_Providers/Tooltip.php [hunk 3]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.8
 	 */
 	public function register() {
-		tribe_singleton( 'tooltip.view', '\Tribe\Tooltip\View' );
+		tribe_singleton( 'tooltip.view', View::class );
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:22]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: b97aad7dc530944716a6181a55950870</sub>

File: src/Tribe/Service_Providers/Tooltip.php (hunk 2 above)
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:24]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: cf5569c56a23cba3a2acb9be547b376b</sub>

File: src/Tribe/Service_Providers/Tooltip.php [hunk 4]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
...
 
-		$this->hook();
+		$this->register_assets();
 	}
 
 	/**
-	 * Setup hooks for classes.
+	 * Registers the assets for the Tooltip class.
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:28]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ed634d45dba135ea8f6bc52542f8ac35</sub>

File: src/Tribe/Service_Providers/Tooltip.php [hunk 5]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
...
 	 *
-	 * @since 4.9.8
+	 * @since TBD
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:30]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 8382ed618054e7fddea6126920a81d45</sub>

File: src/Tribe/Service_Providers/Tooltip.php [hunk 6]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
...
-	}
-
-	/**
-	 * Register assets associated with tooltip
-	 *
-	 * @since 4.9.8
-	 */
-	public function add_tooltip_assets() {
-		$main = \Tribe__Main::instance();
+	public function register_assets() {
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:32]: * :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Public method name "Tooltip::register_assets" is not in camel caps format
* :no_entry_sign: Opening brace should be on a new line

<sub>hash: b1f5b0268f22dba32bf29394189cbcbd</sub>

File: src/Tribe/Service_Providers/Tooltip.php [hunk 7]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
...
+		$main = Common::instance();
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:33]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f48620d95abe6ea232d155d7eb6aaa84</sub>

File: src/Tribe/Service_Providers/Tooltip.php [hunk 8]
```diff
@@ -16,34 +19,25 @@ class Tooltip extends \tad_DI52_ServiceProvider {
...
 
 		tribe_asset(
 			$main,
 			'tribe-tooltip',
 			'tooltip.css',
 			[ 'tribe-common-skeleton-style' ],
-			[ 'wp_enqueue_scripts', 'admin_enqueue_scripts' ],
+			[],
```
Comment (tr1b0t) [inline src/Tribe/Service_Providers/Tooltip.php:40]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: f3ac665175629ea88acf5ebe4bb144b5</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 9 error(s)

**Comment by borkweb** [comment]
This is stale and archived elsewhere. Closing this PR and deleting the branch.

**Pull request closed without merging**